noether_gui/include/noether_gui/widgets/tool_path_modifier_widgets.h

```diff
@@ -0,0 +1,86 @@
+#pragma once
+
+#include <noether_gui/widgets.h>
+
+#include <noether_tpp/core/tool_path_modifier.h>
+
+class QFormLayout;
+class QLabel;
+class QSpinBox;
+
+namespace Ui
+{
+class Vector3dEditor;
+}
+
+namespace noether
+{
+class StandardEdgePathsOrganizationModifierWidget : public ToolPathModifierWidget
+{
+  Q_OBJECT
+public:
+  StandardEdgePathsOrganizationModifierWidget(QWidget* parent = nullptr);
+
+  ToolPathModifier::ConstPtr create() const override;
+
+private:
+  Ui::Vector3dEditor* ui_;
+};
+
+struct RasterOrganizationModifierWidget : public ToolPathModifierWidget
+{
+  Q_OBJECT
+public:
+  using ToolPathModifierWidget::ToolPathModifierWidget;
```

Comment:
What does this `using` statement do?

Reply:
Explicitly let's the class use the base class' constructor